Sources/LanguageServerProtocol/SupportTypes/WorkspaceSettings.swift

```diff
@@ -59,6 +56,11 @@ public struct ClangWorkspaceSettings: Codable, Hashable {
     self.compilationDatabasePath = compilationDatabasePath
     self.compilationDatabaseChanges = compilationDatabaseChanges
   }
+
+  internal func isValid() -> Bool {
```

Comment:
This can be a computed property, making the code a little more swifty.

```suggestion
  var isValid: Bool {
```